substrate-node/pallets/pallet-tfgrid/src/node.rs

```diff
@@ -294,7 +294,6 @@ pub fn validate_longitude_input(input: &[u8]) -> bool {
         }
 }
 
-pub const MIN_SERIAL_NUMBER_LENGTH: u32 = 10;
 pub const MAX_SERIAL_NUMBER_LENGTH: u32 = 50;
```

Comment:
So we keep it to 50 or change it to 128?

Reply:
Right, lets set it to 128

I merged and deployed previous PR because dev was blocked on this. Let's do this in followup pr